ughub.py

```diff
@@ -824,6 +824,67 @@ def InstallPackage(args):
 			print("WARNING: problems were detected during dry installation run. See above.")
 		return
 
+
+def UninstallPackage(args):
+	"""
+	Uninstalls a package
+	:param args: the packages to be removed
+	:return:
+	"""
+	packageNames = args
+
+	for i in range(len(args)):
+		if args[i][0] == "-":
+			packageNames = args[0:i]
+			break
+
+	if len(packageNames) == 0:
+		print("Please specify a package name. See 'ughub help uninstall'.")
+		return
+
+	all_packages = LoadPackageDescs()
+	for package in packageNames:
+		for p in all_packages:
+			if p["name"] == package:
+				print(p["prefix"])
+				if PackageIsInstalled(p) and CheckPackageBeforeUninstall(p):
+					import shutil
+					shutil.rmtree(os.path.join(GetRootDirectory(), os.path.join(p["prefix"], p["name"])))
+
+
+def CheckPackageBeforeUninstall(package):
```

Comment:
Please rename this method to CheckDirForLocalChanges(path) and return 'True' if local changes exist or if git couldn't be executed on that path (e.g. directory not under git control) and 'False' else.

`path` should be a string. The path of a package can be obtained through `GetPackageDir(pkg)`.

Thanks!